ccnmtlsettings/shared.py

```diff
@@ -8,6 +8,7 @@ def common(**kwargs):
     base = kwargs['base']
 
     DEBUG = True
+    TEMPLATE_DEBUG = DEBUG
```

Comment:
The `TEMPLATE_DEBUG` option is deprecated, so I think this line should just be removed. We can still alter this setting in `local_settings.py` via the templates dictionary as needed.